common/utils.go

```diff
@@ -141,3 +141,11 @@ func TruncateWithDotN(s string, n int) string {
 func TruncateWithDot(s string) string {
 	return TruncateWithDotN(s, DefaultTruncateLength)
 }
+
+func StrPtr(s string) *string {
```

Comment:
Could we use generics instead repeating the same implementation for different types?